manifests/init.pp

```diff
@@ -71,7 +71,7 @@
   $mysql_connector_format                                           = 'tar.gz',
   Stdlib::Absolutepath $mysql_connector_install                     = '/opt/MySQL-connector',
   Stdlib::HTTPUrl $mysql_connector_url                              = 'https://dev.mysql.com/get/Downloads/Connector-J',
-  String[1] $mysql_connector_jar_suffix                             = '-bin',
+  String $mysql_connector_jar_suffix                                = '-bin',
```

Comment:
can you please add a test for this that ensures it works with an empty string? Otherwise people might readd `String[1]` in the future.

Reply:
Resolved and tests added - please see comment below